ui/apps/platform/src/Containers/Clusters/ClustersTablePanel.tsx

```diff
@@ -284,7 +286,7 @@ function ClustersTablePanel({
     // After there is a response, if there are no clusters nor search filter:
     if (currentClusters.length === 0 && !hasSearchApplied) {
         return isMoveInitBundlesEnabled ? (
-            <NoClustersPage />
+            <NoClustersPage isModalOpen={isModalOpen} setIsModalOpen={setIsModalOpen} />
```

Comment:
Could one future improvement be to move the modal itself up above conditional rendering, and just past `setIsModalOpen to the <NoClustersPage>, so that there is only one modal, which could be opened from different sub-views?

Reply:
Yes, for sure!